Scripts/Source/minai_AmbientSexTalk.psc

```diff
@@ -0,0 +1,113 @@
+Scriptname minai_AmbientSexTalk extends Quest  
+
+minai_Config config
+SexLabFramework slf = None
+; store registered threads by ostim/sexlab framework:  {ostim: [0,1,2,3], sexlab: [0,1,2,3]}
+int jThreadsByFrameworkMap
+int playerThread = -1
+
+Function OnSexStart(int ThreadID, string framework, minai_Config configLocal, SexLabFramework slfLocal = None)
+    if(!config.enableAmbientComments)
+        return
+    endif
+    MiscUtil.PrintConsole("Ambient:OnSexStart")
+    config = configLocal
+    RegisterForSingleUpdate(config.commentsRate)
```

Comment:
Is this on the main quest? I have a vague recollection that this will trigger the other OnUpdate's on the scripts attached to the same quest.

Reply:
oh really? I thought separate script is enough... Well then I think I'll need to create completely separate quest

Reply:
When you do add a separate quest, pull the latest minai.esp again before making the change. I'll hold off touching the esp until you're done with this.

Reply:
I added new quest to esp, and attached this script there. I followed common pattern with `Maintenance` function, only difference I'm calling my maintenance not from main controller but from minai_Sex.